apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -65,6 +68,8 @@ const {
 const { WRONG_VALUE } = require('@app/condo/domains/common/constants/errors')
 const { get } = require('lodash')
 
+dayjs.extend(duration)
```

Comment:
You should do it once inside index.js. And don't move it from one file to another